e2e/test/scenarios/visualizations-tabular/visualizations-tabular-reproductions.cy.spec.js

```diff
@@ -954,6 +955,51 @@ describe.skip("issue 25415", () => {
   });
 });
 
+describe("issue 7884", () => {
+  const sourceQuestionDetails = {
+    native: {
+      // previously was "SELECT 1 AS C1, 2 AS C2, 3 AS C3"
+      query: "SELECT 1 AS C1, 3 AS C3",
+    },
+  };
+
+  const getNestedQuestionDetails = cardId => ({
+    query: {
+      "source-table": `card__${cardId}`,
+    },
+    display: "table",
+    visualization_settings: {
+      "table.columns": [
+        { name: "C3", enabled: true },
+        { name: "C1", enabled: true },
+        { name: "C2", enabled: true },
```

Comment:
Is this simulating the dropped column?

Reply:
Yes

Reply:
Ok, I get it now.
It's very clever but it's reeeeeeeally not obvious what's happening at the first glance.

Maybe just add a comment on this line instead of rewriting the test with API calls.

Reply:
Updated the test now, please take a look

Reply:
Very clear what's happening now.
But once again - the original idea was so clever. I wish I'd thought of it in some of the tests. :D

Does it matter that we're not visiting questions when we create them using the API?
Does that only affect the `result_metadata`?

Reply:
Confirmed that the updated test is failing on `master` branch without this fix.
All good.